app/src/org/odk/collect/android/views/media/MediaLayout.java

```diff
@@ -159,7 +168,68 @@ public void onClick(View v) {
         String errorMsg = null;
 
         View imageView= null;
-        if(qrCodeContent != null ) {
+        if(inlineVideoURI != null) {
+            String error = null;
+            try {
+                final String imageFilename = ReferenceManager._().DeriveReference(inlineVideoURI).getLocalURI();
+
+//                MediaPlayer mp = new MediaPlayer();
+//                mp.setDataSource(imageFilename);
+//                mp.prepare();
+//
+//                int preferredVideoWidth = mp.getVideoWidth();
+//                int preferredVideoHeight = mp.getVideoHeight();
+
+                //Not sure if this is useful yet
+                int[] maxBounds = getMaxImageViewBounds();
```

Comment:
Just want to make sure that you want to get the max image view before setting up ```mVideoView```, which seems like it might change the result of ```getMaxImageViewBounds```